### PR TITLE
Return sets of authors instead of one author during parsings pages wi…

### PR DESCRIPTION
**Pull request by sledzgrzegorz**
Return sets of authors instead of one author during parsings pages with promotions

### CODE REVIEW

File: parser/src/test/java/parser/empik/EmpikBookCrawlerTest.java [hunk 1]
```diff
@@ -33,15 +34,18 @@ public void testGetBooks() throws Exception {
...
 
         //act
         Set<BookDetails> books=bookCrawler.getBooks(urlsToBookDetails);
         //assert
         for(BookDetails resultBook: books) {
             SoftAssert sf=new SoftAssert();
             sf.assertEquals(books.size(),1);
-            sf.assertEquals(expectedBook.getAuthor(),resultBook.getAuthor());
+            sf.assertEquals(expectedBook.getAuthors(),resultBook.getAuthors());
```
Comment (ghost) [inline parser/src/test/java/parser/empik/EmpikBookCrawlerTest.java]: The order of arguments should be changed :octopus:

Comment (caroo7) [reply parser/src/test/java/parser/empik/EmpikBookCrawlerTest.java]: :neckbeard:

Comment (sledzgrzegorz) [reply parser/src/test/java/parser/empik/EmpikBookCrawlerTest.java]: Nice suggestion :+1:

**Comment by ghost** [comment]
Good job, now go to sleep :zzz:  :+1:

**Pull request merged**